uaclient/status.py

```diff
@@ -198,8 +198,8 @@ def format_tabular(status: "Dict[str, Any]") -> str:
         content = [
             STATUS_UNATTACHED_TMPL.format(
                 name="\nSERVICE",
-                available="AVAILABLE",
-                description="DESCRIPTION",
+                available=" AVAILABLE",
```

Comment:
Is the issue here actually that we have `\n` at the start of name, which takes up one "character" of the line?

Reply:
absolutely right. good suggestion. I've added a unit test and fix the change properly.